tests/npm-app/build_bundle.js

```diff
@@ -0,0 +1,49 @@
+import { readdirSync, statSync, readFileSync, writeFileSync } from "fs";
```

Comment:
@letmaik - Could you please scan this file, and let me know if it does anything egregious? It's taking a CCF JS app `src` directory, and converting that into a `bundle` object ready to be inserted into `set_js_app` proposal. My JS is very rusty, so all of the library functions are the first thing that worked, please let me know if there's obvious alternatives.

Reply:
Wouldn't it be nice to ship this script as part of the `ccf-app` npm package? Then we could document its usage rather than explaining to users how to include the content of `app.json` in the `set_js_app` proposal?

Reply:
I don't think we want to increase the number of CLI tools we ship and support, or the number of languages we do it in. I think we especially want to avoid doing that in JS, considering the dependency climate in the npm world. I suggest we use the ccf-app package strictly to ship TypeScript interfaces, unless there's material user demand to the contrary that we think we have the resources to provide.

Reply:
As discussed - we want to make a distinction between our fully-maintained API and helpful samples. This is, for now, fully in the latter camp.